rest_framework-stubs/serializers.pyi

```diff
@@ -213,21 +204,22 @@ class ListSerializer(
         allow_null: bool = ...,
     ): ...
     def get_initial(self) -> List[Mapping[Any, Any]]: ...
-    def validate(self, attrs: OrderedDict) -> OrderedDict: ...
+    def validate(self, attrs: Any) -> Any: ...
     @property
     def data(self) -> ReturnList: ...
     @property
     def errors(self) -> ReturnList: ...
 
 def raise_errors_on_nested_writes(method_name: str, serializer: BaseSerializer, validated_data: Any) -> NoReturn: ...
 
-class ModelSerializer(Serializer, BaseSerializer[_MT, Mapping[str, Any], Dict[str, Any], _MT]):
+class ModelSerializer(Serializer, BaseSerializer[_MT]):
```

Comment:
Let's add a bunch of tests for this type! It would be super helpful in the future.

Reply:
No time now...